Formula/xcinfo.rb

```diff
@@ -0,0 +1,20 @@
+class Xcinfo < Formula
+  desc "Tool to get information about and install available Xcode versions"
+  homepage "https://github.com/xcodereleases/xcinfo"
+  url "https://github.com/xcodereleases/xcinfo/archive/0.5.0.tar.gz"
+  sha256 "07b6c2f5a5b6a5f3cd6d190ac16c36c5011a87f2a1da2a109ccaa909eb6bcc2c"
+  license "MIT"
+
+  depends_on xcode: "12.0"
+
+  def install
+    system "swift", "build",
+           "--configuration", "release",
+           "--disable-sandbox"
+    bin.install ".build/release/xcinfo"
+  end
+
+  test do
+    system "true"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.